include/kernels/dslash_pack.cuh

```diff
@@ -217,12 +226,16 @@ namespace quda
 
     if (face_num == 0) { // backwards
       int idx = indexFromFaceIndexStaggered<4, QUDA_4D_PC, dim, nFace, 0>(ghost_idx, parity, arg);
-      Vector f = arg.in[src_idx](idx, spinor_parity);
-      arg.halo_pack.Ghost(dim, 0, ghost_idx + src_idx * nFace * arg.dc.ghostFaceCB[dim], spinor_parity) = f;
+      for (auto s = 0; s < n_src_tile; s++) {
```

Comment:
Is there a specific reason why you opted for `for` loop from `0 -> n_src_tile` here as opposed to `src -> src + n_src_tile` in `pack`? I think the convention should be consistent between the routines; I have no particular preference for which convention gets used.